### ISSUE TITLE
[job failure]ci-kubernetes-e2e-gci-gke|gce-serial

### ISSUE THREAD

**Issue by krzyzacy** [kubernetes/kubernetes#60003]
/priority failing-test
/kind bug
/status approved-for-milestone
/sig gcp

https://k8s-testgrid.appspot.com/sig-release-master-blocking#gci-gce-serial
https://k8s-testgrid.appspot.com/sig-release-master-blocking#gci-gke-serial

the jobs are on the [master-blocking dashboard](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and seems borked from yesterday.

cc @jdumars @abgworrall @jberkus

**Comment by krzyzacy** [comment]
now looks like it's failing on `Services should work after restarting apiserver`?
/sig network
cc @MrHohn @bowei

**Comment by krzyzacy** [comment]
and now `Daemon set` suite is also failing on https://k8s-testgrid.appspot.com/sig-release-master-blocking#gci-gce-serial
/sig apps

**Comment by kow3ns** [comment]
triaged. I'll issue a PR to fix this.

**Comment by krzyzacy** [comment]
@kow3ns any update?

**Comment by kow3ns** [comment]
The DaemonSet tests are green. We need to find owners from sig-network for the remaining serial tests.
Also, we need to update the test grid. The tests are now considered to be conformance.

**Comment by krzyzacy** [comment]
gce-serial is not very stable, I'll update the issue after a few runs

**Comment by krzyzacy** [comment]
Seems there are multiple failing/flaky tests, I'll open a few more issues and leave this one open as an umbrella issue until two suites becomes fully green.

**Comment by janetkuo** [comment]
Fixes for two DaemonSet test failures: #60747, #60820

**Comment by krzyzacy** [comment]
no they are not, not until all the linked test failures are closed

**Comment by janetkuo** [comment]
All DaemonSet tests are passing. Removing sig/apps and unassigning sig-apps owners. IMO we should file one bug for each failing test.

**Comment by krzyzacy** [comment]
@janetkuo yep I did that, see all linked individual test failure issues above.

**Comment by krzyzacy** [comment]
#60763 and #60762 left here

**Comment by krzyzacy** [comment]
https://k8s-testgrid.appspot.com/sig-release-master-blocking#gci-gke-serial is green now, pending on fix for https://github.com/kubernetes/kubernetes/issues/60763

/sig apps
